dev-util/geany-plugins/geany-plugins-1.34.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Did you try with 7? Looks to me like it can be used here. Also one empty line below EAPI please.

Reply:
7 works - updated. Additional line added.